app/src/main/res/layout/fragment_client_settings.xml

```diff
@@ -9,7 +9,7 @@
     android:focusableInTouchMode="true"
     android:background="@color/primary_dark">
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/3c5401c2c1e764397645482fac20357348df064f/app/src/main/res/layout/fragment_client_settings.xml#L10"></span>Possible overdraw: Root element paints background <code>@color/primary_dark</code> with a theme that also paints a background (inferred theme is <code>@style/AppTheme</code>)</td>
    </tr>
  </tbody>
</table>